brave/browser-laptop
# add sync QR code and setup options

This adds most of the UX for Brave sync setup in the case where this device is the first device in the sync profile.

Auditors: @ayumi

Test Plan:
1. npm run clean-session-store
2. npm start
3. go to about:preferences#sync
4. click the orange button to set up sync for the first time
5. name the device, then click the create button
6. shut down brave, run npm start again
7. you should see the device name from (5) in the terminal console
8. go to about:preferences#sync, click 'add new device' button
9. you should see a QR code

![screen shot 2017-01-13 at 8 15 43 am](https://cloud.githubusercontent.com/assets/549654/21923580/ca39bd9c-d96c-11e6-86b5-d7cabbf4bf12.png)
![screen shot 2017-01-13 at 8 14 29 am](https://cloud.githubusercontent.com/assets/549654/21923586/d2941d66-d96c-11e6-9c2d-abd5ede493c5.png)
